src/portable/raspberrypi/rp2040/rp2040_usb.h

```diff
@@ -71,11 +71,13 @@ struct hw_endpoint
     uint len;
     // Amount of data with the hardware
     uint transfer_size;
+#ifdef RP2040_USB_HOST_MODE
```

Comment:
if the field order is not matter, we should merge this with line 89 into a single `#ifdef`